ontology/ontocompany/OntoCompany.csv

```diff
@@ -40,6 +40,16 @@ hasNumberofManufacturingFacilities,Data Property,hasCompanyProperty,IS-A,Company
 hasYearOfEstablishment,"Data Property, Functional Property",hasCompanyProperty,IS-A,Company,https://www.w3.org/TR/xmlschema11-2/#gYear,,A relationship between a company and its year of formation.,http://www.theworldavatar.com/kg/ontocompany/,Year of Establishment
 hasSectoralDescription,Data Property,hasCompanyProperty,IS-A,Company,String,,A relationship between a company and its sectoral description.,http://www.theworldavatar.com/kg/ontocompany/,Sectoral Description
 hasSSICCode,Data Property,hasCompanyProperty,IS-A,Company,Integer,,A relationship between a company and its SSIC code corresponding to sectoral description (Singapore specific).,http://www.theworldavatar.com/kg/ontocompany/,SSIC Code
+isLocatedIn,Object Property,,,Company,http://www.opengis.net/citygml/building/2.0/Building,,Links a company to the building where it is located.,http://www.theworldavatar.com/kg/ontocompany/,is Located In
+hasUniqueEntityNumberStatus,Data Property,hasCompanyProperty,IS-A,Company,String,exactly 1,A relationship between a company and its Unique Entity Number status.,http://www.theworldavatar.com/kg/ontocompany/,has Unique Entity Number Status
+hasCompanyType,Data Property,hasCompanyProperty,IS-A,Company,String,,A relationship between a company and its Company type.,http://www.theworldavatar.com/kg/ontocompany/,has Company Type
+hasUniqueEntityNumber,Object Property,hasCompanyProperty,IS-A,Company,UniqueEntityNumber,exactly 1,A relationship between a company and its Unique Entity Number.,http://www.theworldavatar.com/kg/ontocompany/,has Unique Entity Number
+UniqueEntityNumber,Class,,,,,,"The Unique Entity Number (UEN) is a standard identification number for entities, ranging from businesses and partnerships to charities and societies, to interact with government agencies in Singapore.",http://www.theworldavatar.com/kg/ontocompany/,Unique Entity Number
+FormalOrganization,Class,,,,,,"An organization that is recognized in some legal jurisdiction with associated rights and responsibilities",http://www.theworldavatar.com/kg/ontocompany/,Formal Organization
+hasIdentifierValue,Data Property,,,UniqueEntityNumber,String,,A relationship between a Unique Entity Number and its literals.,http://www.theworldavatar.com/kg/ontocompany/,has Identifier Value
+isIssuedBy,Object Property,,,UniqueEntityNumber,FormalOrganization,,A relationship between a Unique Entity Number and its issuing organisation.,http://www.theworldavatar.com/kg/ontocompany/,is Issued By
+hasIssueDate,Object Property,http://www.w3.org/2006/time#hasTime,IS-A,UniqueEntityNumber,https://www.w3.org/TR/owl-time/#Instant,,A relationship between a Unique Entity Number and its date of issue.,http://www.theworldavatar.com/kg/ontocompany/,has Issue Date
```

Comment:
domain and range for hasIssueDate is not necessary because it inherits from time:hasTime